Add blog post for responsive_ux feature rollout (#277)
openSUSE/obs-landing: _posts/development/2021-01-12-responsive-ux-rollout.md

---
layout: post
title: New Navigation Design Ready to Come Out!
category: development
---

After a few months as part of the [Beta program](https://openbuildservice.org/2018/10/04/the-beta-program),
the new navigation areas are ready to be rolled out for every single user in OBS.
Framed in an improved responsive layout, the new navigation design will appear in the OBS interface soon.

> These two images summarize this milestone:

Reviewer: ```suggestion
As we say, a picture is worth a thousand words:
```

 <figure>
  <img src="/images/posts/sprint_90_responsive_ux_rollout_before.png" alt="Home page with old navigation">
  <figcaption>Home page with old navigation</figcaption>
</figure>

<figure>
  <img src="/images/posts/sprint_90_responsive_ux_rollout_after.png" alt="Home page with new navigation design and other changes in the layout">
  <figcaption>Home page with new navigation design and other changes in the layout</figcaption>
</figure>

> The navigation areas make the main difference. The top bar includes options closely related to the user like the Profile or his/her Watchlist.

Reviewer: ```suggestion
The redesign introduces changes mostly in the navigation areas. The top bar includes links closely related to the user like their profile or their watchlist.
```

> The left sidebar contains links to both relevant places inside the web application and general actions.

Reviewer: ```suggestion
The left sidebar contains links to both relevant places inside OBS and actions on the current page.
```

Moreover, we encourage you to use OBS from your tablet or smartphone, the layout and the navigation areas have also been revamped to offer
the best user experience possible for small screens.

<figure>
  <img src="/images/posts/sprint_90_responsive_ux_rollout_small_after.png" alt="Home page with new navigation design for small screens">
  <figcaption>Home page with new navigation design for small screens</figcaption>
</figure>

> Thanks to the valuable feedback provided by the users in our Beta program, we have come up with this solution that tries to provide a more

Reviewer: ```suggestion
Thanks to the valuable feedback provided by the users in our beta program, we have come up with this solution that tries to provide a more
```

user-friendly and intuitive interface for OBS.
We invite you to have a look at the blog posts we published from the beginning of this journey to nowadays:

- [More Responsive Than Ever Before!](https://openbuildservice.org/2020/02/17/mobile-first-layout)
- [Who Said OBS Was Not Responsive?](https://openbuildservice.org/2020/03/03/who-said-obs-was-unusable)
- [Fresh Notifications and Much More](https://openbuildservice.org/2020/03/24/notifications-redesign)
- [Where Are My Package Actions?](https://openbuildservice.org/2020/05/18/quick-update-about-package-actions)
- [Left Side Navigation](https://openbuildservice.org/2020/08/17/left-side-navigation)
- [Navigation and Profile Page Redesign](https://openbuildservice.org/2020/10/22/navigation-and-profile-page) section _"To Collapse, or Not To Collapse?"_.
- [Revisit Actions in the Left Side Navigation](https://openbuildservice.org/2020/12/03/revisit-actions-left-side-navigation)

> As you can see in the blog entries and screenshots, this is not all about navigation. You can perceive many other differences in the redesigned responsive layout. You will find the following changes, among others:

Reviewer: ```suggestion
As you can see in the blog entries and screenshots, this is not all about navigation. You will find, among others, the following changes in the redesigned responsive layout:
```

---
layout: post
title: New navigation design ready to come out!
category: development
---

After a few months as part of the [Beta program](https://openbuildservice.org/2018/10/04/the-beta-program),
the new navigation areas are ready to be rolled out for every single user in OBS. Together with some other improvements
in the layout and a complete adaptation for responsiveness, the new navigation design will appear in the OBS interface soon.

These two images summarize this milestone:

 <figure>
  <img src="/images/posts/sprint_90_responsive_ux_rollout_before.png" alt="Home page with all navigation and without responsiveness">

>   <figcaption>Home page with all navigation and without responsiveness</figcaption>

Reviewer 2: @saraycp what do you mean by `with all navigation`?

Author: ups! I meant "old". Changing....

Reviewer 2: perfect

</figure>

<figure>
  <img src="/images/posts/responsive_ux_rollout_after.png" alt="Home page with new navigation design and adjustments for responsiveness">
  <figcaption>Home page with new navigation design and adjustments for responsiveness</figcaption>
</figure>

The navigation areas make the main difference. The top bar includes options closely related to the user like the Profile or his/her Watchlist.
The left sidebar contains links to both relevant places inside the web application and general actions.

> But more important than that is the work we have done to convert our interface into a responsive one.

Reviewer 2: Not sure if its more important, I would suggest to go more into direction "another important part"

Author: "As important as that" ? ;)

Reviewer 3: What about `But let's not forget about the work we ...`?

Reviewer 3: Or better, I would remove the whole sentence, the rest of that paragraph already conveys the idea.

Reviewer 2: I'm fine with both, I just thought its maybe a bit to much to say its more important :+1:

Author: I rephrased the paragraph. What do you think now?

Reviewer 2: :+1:

for small screens.

<figure>
  <img src="/images/posts/responsive_ux_rollout_small_after.png" alt="Home page with new navigation design and adjustments for responsiveness in small screens">
  <figcaption>Home page with new navigation design and adjustments for responsiveness in small screens</figcaption>
</figure>

Thanks to the valuable feedback provided by the users in our Beta program, we have come up with this solution that tries to provide a more
user-friendly and intuitive interface for OBS.
We invite you to have a look at the blog posts we published from the beginning of this journey to nowadays:

- https://openbuildservice.org/2020/02/17/mobile-first-layout
- https://openbuildservice.org/2020/03/03/who-said-obs-was-unusable
- https://openbuildservice.org/2020/03/24/notifications-redesign
- https://openbuildservice.org/2020/05/18/quick-update-about-package-actions
- https://openbuildservice.org/2020/08/17/left-side-navigation
- https://openbuildservice.org/2020/10/22/navigation-and-profile-page section "To Collapse, or Not To Collapse?".

> - https://openbuildservice.org/2020/12/03/revisit-actions-left-side-navigation

Reviewer 3: I believe this should be clickable urls

Author: True!

Author: Ok, I've used the real titles for the texts of the links. They look much better now.

As you can see in the blog entries, this is not all about navigation. You will notice the following changes, among others:

- Responsive Build Results area.
- Replace forms in modals by inline forms.
- Responsive tables.
- Footer redesign.
- Sponsors at the bottom of the page.

> - In-place edition.

Reviewer 2: ```suggestion
- In-place editing.
```

---
layout: post
title: New navigation design ready to come out!
category: development
---

After a few months as part of the [Beta program](https://openbuildservice.org/2018/10/04/the-beta-program),
the new navigation areas are ready to be rolled out for every single user in OBS. Together with some other improvements
in the layout and a complete adaptation for responsiveness, the new navigation design will appear in the OBS interface

> during the upcoming hours or days.

Reviewer 2: @saraycp I wouldn't write `hours or days`, maybe just write that it will be rolled out after this blog post is published. To warn the user we can use the status messages and mailing list

Author: I just simplified the sentence, let me know what you think.

Reviewer 2: I like it :+1:

> during the upcoming hours or days.

Author: This sentence was changed a few "pushes" ago :smiley:

 These two images summarize this milestone:

 <figure>
  <img src="/images/posts/responsive_ux_rollout_before.png" alt="Home page with all navigation and without responsiveness">
  <figcaption>Home page with all navigation and without responsiveness</figcaption>
</figure>

<figure>
  <img src="/images/posts/responsive_ux_rollout_after.png" alt="Home page with new navigation design and adjustments for responsiveness">
  <figcaption>Home page with new navigation design and adjustments for responsiveness</figcaption>
</figure>

The navigation areas make the main difference. The top bar includes options closely related to the user like the Profile or his/her Watchlist.
The left sidebar contains links to both relevant places inside the web application and general actions.

But more important than that is the work we have done to convert our interface into a responsive one.

> We encourage you to surf our website from your tablet or smartphone. You will notice the improvements at a glance.

Reviewer 4: suggestion:
"to surf OBS" instead of "to surf our website"?

Author: done!

> We encourage you to surf our website from your tablet or smartphone. You will notice the improvements at a glance.

Author: done
